Gem/Code/Source/Spawners/RoundRobinSpawner.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of
+ * this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Source/Spawners/IPlayerSpawner.h>
+
+namespace AzFramework
+{
+    struct PlayerConnectionConfig;
+}
+
+namespace Multiplayer
+{
+    struct EntityReplicationData;
+    using ReplicationSet = AZStd::map<ConstNetworkEntityHandle, EntityReplicationData>;
+} // namespace Multiplayer
+
+namespace MultiplayerSample
+{
+    class RoundRobinSpawner
```

Comment:
When is this created? I'm curious how the dependency order works between RoundRobinSpawner and NetworkPlayerSpawnerComponent.

Reply:
Activation of MPSampleSystemComponent. It should precede NetworkPlayerSpawnerComponent Activation reliably.